pal_event_server/lib/src/loggers/app_usage/macos/apple_script_util.dart

```diff
@@ -57,14 +58,9 @@ const scriptArgs = [
   "-e", r'end if',
   "-e", r'else',
   "-e", r'set windowTitle to "__"',
+  "-e", r'set urlText to ""',
```

Comment:
where is urlText consumed?

Reply:
It seems not used currently. Should we log it to Paco server or just remove it?

Reply:
It is being parsed out in the pal_event_helper I found. So, looks good.